examples/config/config

```diff
@@ -312,8 +320,8 @@ set history_disable_easter_egg 1
 @cbind  <Ctrl>p = hardcopy page
 
 # Web searching binds
-@cbind  gg<Google:>_         = uri http://www.google.com/search?q=\@<encodeURIComponent(%r)>\@
-@cbind  ddg<DuckDuckGo:>_    = uri http://duckduckgo.com/?q=%s
+# NOTE you may search Google by adding "!g" to your DuckDuckGo search string
+@cbind  d<DuckDuckGo:>_      = uri http://duckduckgo.com/?q=%s
```

Comment:
I don't think a few extra keys is that bad for the search command.  I think `<Ctrl>/` could work nice another option would be to go with `<Ctrl>k` which is what firefox uses.

As for dropping google I don''t think that's a big deal and t's just weird to have two different websearch bindings in the default config anyway.